src/plugins/intel_npu/src/al/include/intel_npu/config/npuw.hpp

```diff
@@ -149,6 +149,7 @@ DEFINE_OPT(NPUW_LLM_CACHE_ROPE, bool, true, npuw::llm::cache_rope, RunTime);
 DEFINE_OPT(NPUW_LLM_PREFILL_CHUNK_SIZE, uint64_t, 1024, npuw::llm::prefill_chunk_size, RunTime);
 DEFINE_OPT(NPUW_LLM_SHARED_HEAD, bool, true, npuw::llm::shared_lm_head, RunTime);
 DEFINE_OPT(NPUW_LLM_MAX_LORA_RANK, uint32_t, 32, npuw::llm::max_lora_rank, RunTime);
+DEFINE_OPT(NPUW_LLM_OPTIMIZE_FP8E4M3, bool, false, npuw::llm::optimize_fp8, RunTime);
```

Comment:
Hi! Could I please ask you to add this property to the caching properties as done in Pyramid PR, for example: https://github.com/openvinotoolkit/openvino/pull/31917/files#diff-71dbfa5cac64c0b8773a1237f4d5262a543b5dc676d4c93bcbe0bf278144da58R94